packages/hooks/src/useCloseDerivAccount.ts

```diff
@@ -0,0 +1,15 @@
+import { useRequest } from '@deriv/api';
+
+const useCloseDerivAccount = () => {
+    const { data, mutate, error, isError, isSuccess, isLoading } = useRequest('account_closure');
```

Comment:
I think we can export the whole object (mutate fn, data, all states) that is returned by useRequest hook? So it becomes more flexible on which status the dev wants to use

Reply:
**Nit**: Can u add a TSDoc describing what this is used for?